__tests__/app.test.js

```diff
@@ -467,4 +465,50 @@ describe("PATCH /api/articles/:articleid", () => {
   });
 });
 
+describe("DELETE api/comments/:comment_id", () => {
+  test("should return 200", () => {
+    return request(app).delete("/api/comments/1")
+      .expect(204)
+  });
+  test("should delete the comment of the associated comment id", () => {
```

Comment:
This test is not necessary as you have a test for delete already. If that passes, the delete was successful.